show/bgp_common.py

```diff
@@ -152,13 +158,7 @@ def print_ip_routes(route_info, filter_by_ip):
                     if "directlyConnected" in info[i]['nexthops'][j]:
                         print("  * directly connected, {}\n".format(info[i]['nexthops'][j]['interfaceName']))
                     else:
-                        if "ip" in info[i]['nexthops'][j]:
-                            str_2_print = "  * {}".format(info[i]['nexthops'][j]['ip'])
-
-                        if "active" in info[i]['nexthops'][j]:
-                            str_2_print += ", via {}".format(info[i]['nexthops'][j]['interfaceName'])
-                        else:
-                            str_2_print += " inactive"
+                        str_2_print = get_nexthop_info_str(info[i]['nexthops'][j], 1)
```

Comment:
Could use True/False instead of passing 1/0

Reply:
Agreed. Will make the change.  Please see my next commit update for this.